tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCoverageLevel.java

```diff
@@ -1280,7 +1278,7 @@ public void testLSR() {
             }
         }
         if (!assertEquals(
-                "Language subtags that are in a CLDR locale's ID are in root ("
+                "$language in attributeValueValidity.xml needs to include these languages: ("
```

Comment:
This is misleading in a different way. The error just indicates that there is a mismatch.
```suggestion
                "There is a mismatch: each language in the following list either needs to be removed from root.xml, OR added to $language in attributeValueValidity.xml: ("
```

Reply:
What does "removed from root.xml" mean? In the reproducer, "lld" wasn't in root.xml.

Reply:
@macchiati any clarification on this?

Reply:
Hmmm. Why was lld failing; was it for not being in validity?

Reply:
Correct. lld had not been added to validity element in attributeValueValidity.xml

Reply:
#3666 changes this wording to: `"Language subtags in a locale's ID must be in one of the attributeValueValidity.xml $language* sets, typically $languageNonTcLtBasic  ("` which is even clearer. I'd also recommend dropping the message change from this PR.

Reply:
This PR no longer changes the message.